### ISSUE TITLE
No easy to way to check for access token

### ISSUE THREAD

**Issue by GoogleCodeExporter** [geniousprogrammer/socialauth#340]
```
Hi I am using latest version of social auth for server and for Android.

I have this segment of source code on server side (in Scala):

socialAuthConfig.load(inputStreamOption.get)

          val socialAuthManager = new SocialAuthManager

          socialAuthManager.setSocialAuthConfig(socialAuthConfig)

          val providerId = "linkedin"

          val linkedinAccessToken = "36cc6f17-a8e3-4dab-8438-575cf6cdc8a0"

          //val accessGrant = socialAuthManager.createAccessGrant(providerId, linkedinAccessToken, "")
          val provider_config = socialAuthConfig.getProviderConfig(providerId)

          val accessGrant = new AccessGrant(linkedinAccessToken, provider_config.get_consumerSecret())

          //val authProvider = socialAuthManager.getProvider(providerId)

        val authProvider = new LinkedInImpl()

          authProvider.getProviderId

          //Option(authProvider) must beNone


I have not mastered all the differences between consumer key, consumer secret, 
api key, api secret, access token. And also there is some confusion about them.

My main goal is simply to force an access token (which I have already retrieve 
from another source) and verify in server if this is correct.

In other words I believe a method like this one would be really useful:
socialAuthManager.verifyAccessToken(providerId: String, accessToken: String)


Thank you
```

Original issue reported on code.google.com by `pligor.g...@gmail.com` on 30 Apr 2014 at 11:17
